docs/gestor_dependencias.md

```diff
@@ -0,0 +1,11 @@
+# Gestor de dependencias
```

Comment:
Que nadie ha pedido que justifiquéis el gestor de dependencias. Y cuando hay tantos errores que coinciden con otros errores, no suele ser una indicación de que se estén siguiendo las buenas prácticas.

Reply:
Es preferible entonces que borre el documento?

Reply:
Es preferible que sigas las buenas prácticas.